tic_tac_toe/game.py

```diff
@@ -24,18 +28,31 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
-
+    if isinstance(position,int):
```

Comment:
That won't actually work on python3.
```
   x = 'hi'
   x[0] > 1
Traceback (most recent call last):
  File "python", line 1, in <module>
TypeError: unorderable types: str() > int()
```
As for the tests there's a limit to how many edge cases we should be testing for. You could conceivably write an if statement for every single bit of test data we could toss in, but it would still be missing the point. Using `isinstance(position, tuple)` would catch anything we could toss that isn't a tuple(or a class that extends tuple) and doesn't rely on getting lucky with the data.